com.unity.netcode.adapter.utp/Tests/Runtime/ConnectionTests.cs

```diff
@@ -141,8 +154,8 @@ public IEnumerator ServerDisconnectMultipleClients()
             yield return new WaitForSeconds(MaxNetworkEventWaitTime);
 
             // Check that all clients got a Disconnect event.
-            Assert.True(m_ClientsEvents.All(evs => evs.Count == 2));
-            Assert.True(m_ClientsEvents.All(evs => evs[1].Type == NetworkEvent.Disconnect));
+            Assert.That(m_ClientsEvents.All(evs => evs.Count == 2));
```

Comment:
Sorry, I should have been more precise with that comment. If you straight up replace True with That, it will behave exactly the same.

To get relevant error messages, you need to use it in the following way:
`Assert.That(m_ClientEvents, Has.All.EqualTo(your constraint here);`

That way, it will be able to mark which specific collection items are not matching your constraint, and should also display the actual value vs the expected value.

Reply:
Has.All.EqualTo, or any other Has.All method that you prefer

Reply:
Ah right, I see. In that case I'm not sure it's really worth it in that particular case. While the first check (`events.All(evs => evs.Count == 2)`) can be expressed in that style, it's immediately followed by another similar check that I'm not sure can be written using such constraints. I prefer the extra consistency here of having all similar assertions using a similar style.